README.md

```diff
@@ -1,43 +1,99 @@
+## Install
+
+1. Clone and enter this repository:
+  * `git clone https://github.com/mattermost/mattermost-plugin-ai && cd mattermost-plugin-ai`
+2. Start the services: `docker compose up -d`
+3. Configure the Mattermost server from the init script: `bash ./init.sh`
+4. Access Mattermost:
+  * Open Mattermost at `http://localhost:8065`
+  * Select **View in Browser**
+  * Log in with the generated `root` credentials
+5. Install the `mattermost-plugin-ai` on Mattermost:
+  * Go the releases page and download the latest release.
+  * In the top left Mattermost menu, click **System Console** ➡️ **Plugin Management** and [upload the plugin to install it]((https://docs.mattermost.com/administration/plugins.html#plugin-uploads))
+  * Enable the plugin and configure plugin settings as desired.
+
+Lots of unfinished work in the system console settings. For now all you need to do is input and OpenAI API Key and configure allowed teams/users as desired. More options and the ability to use local LLMs is on the roadmap.
+
+## Usage
+
+### Conversation
 
 Chat with an LLM right inside the Mattermost interface. Answer are streamed so you don't have to wait:
 
 https://github.com/mattermost/mattermost-plugin-ai/assets/3191642/f375f1a2-61bf-4ae1-839b-07e44461809b
 
-## Thread Summarization
+### Thread Summarization
 Use the post menu or the `/summarize` command to get a summary of the thread in a DM:
+
 ![Summarizing Thread](/img/summarize_thread.png)
 
-## Answer questions about Threads
+### Answer questions about Threads
 Respond to the bot post to ask follow up questions:
 
 https://github.com/mattermost/mattermost-plugin-ai/assets/3191642/6fed05e2-ee68-40db-9ee4-870c61ccf5dd
 
-## Chat anywhere
+### Chat anywhere
 Just mention @llmbot anywhere in Mattermost to ask it to respond. It will be given the context of the thread you are participating in:
+
 ![Bot Chat](/img/mention_bot.png)
 
-## React for me
+### React for me
 Just for fun! Use the post menu to ask the bot to react to the post. It will try to pick an appropriate reaction.
 
 https://github.com/mattermost/mattermost-plugin-ai/assets/3191642/5282b066-86b5-478d-ae10-57c3cb3ba038
 
-## RLHF Feedback Collection
-Bot posts will have :+1: :-1: icons for collecting feedback. The idea would be to use this as input for RLHF fine tuning.
+### RLHF Feedback Collection
+Bot posts will have 👍 👎 icons for collecting feedback. The idea would be to use this as input for RLHF fine tuning.
 
+## Related Efforts
 
-## Installation
+Explore Mattermost's AI initiatives:
 
-1. Go the releases page and download the latest release.
-2. On your Mattermost, go to System Console -> Plugin Management and upload it. [More Details](https://docs.mattermost.com/administration/plugins.html#plugin-uploads)
-3. Enable the plugin and configure plugin settings as desired.
+* https://mattermost.github.io/mattermost-ai-site/
+* https://community.mattermost.com/core/channels/ai-exchange
+* https://forum.mattermost.com/c/ai-frameworks/40
```

Comment:
We could enable discussions on this repo instead? What do you think?

Reply:
Definitely, I think we can just swap out these links in the coming days as we organize our content.